tests/conftest.py

```diff
@@ -0,0 +1,142 @@
+import logging
+import os
+import pathlib
+from contextlib import nullcontext
+from pathlib import Path
+
+import datajoint as dj
+import pytest
+from element_interface.utils import QuietStdOut, find_full_path, value_to_bool
+
+from workflow_zstack.paths import get_volume_root_data_dir
+
+# ------------------- SOME CONSTANTS -------------------
+
+
+logger = logging.getLogger("datajoint")
+
+
+sessions_dirs = [
+    "subject1",
+]
+
+
+
+
+@pytest.fixture(autouse=True, scope="session")
+def setup(request):
+    """Take passed commandline variables, set as global"""
+    global verbose, _tear_down, test_user_data_dir, verbose_context
+
+    verbose = value_to_bool(request.config.getoption("--dj-verbose"))
+    _tear_down = value_to_bool(request.config.getoption("--dj-teardown"))
+    test_user_data_dir = Path(request.config.getoption("--dj-datadir"))
+    test_user_data_dir.mkdir(exist_ok=True)
+
+    if not verbose:
+        logging.getLogger("datajoint").setLevel(logging.CRITICAL)
+
+    verbose_context = nullcontext() if verbose else QuietStdOut()
+
+    yield verbose_context, verbose
+
+
+# --------------------  HELPER CLASS --------------------
+
+
+def null_function(*args, **kwargs):
+    pass
+
```

Comment:
That function was used in the components of pytests that are no longer relevant when running in Dev Containers. It will be removed in the next commit.